docker-compose.yaml

```diff
@@ -15,9 +15,10 @@ services:
       FEATURES_BATCH_DELAY: "${FEATURES_BATCH_DELAY}"
       FEATURES_UNIT_DELAY: "${FEATURES_UNIT_DELAY}"
       FEATURES_CURRENT_PLAYERS_UPDATE_INTERVAL_DELAY: "${FEATURES_CURRENT_PLAYERS_UPDATE_INTERVAL_DELAY}"
+      FEATURES_RELEASE_DATE_UPDATE_INTERVAL_DELAY: "${FEATURES_RELEASE_DATE_UPDATE_INTERVAL_DELAY}"
       FEATURES_UPDATE_INTERVAL_DELAY: "${FEATURES_UPDATE_INTERVAL_DELAY}"
       FEATURES_ITERATION_DELAY: "${FEATURES_ITERATION_DELAY}"
-      RUNNER_ITERATION_DELAY: "${RUNNER_ITERATION_DELAY}"
+      RUNNER_DEFAULT_ITERATION_DELAY: "${FEATURES_ITERATION_DELAY}"
```

Comment:
unused env var: RUNNER_DEFAULT_ITERATION_DELAY

Switche to RUNNER_DEFAULT_ITERATION_DELAY for the default delay. Remove FEATURES_ITERATION_DELAY. Also update the config.loader accordingly.